packages/preview/ctu-report/0.2.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ctu-report"
```

Comment:
If this is an official package, a member of the CTU staff should send an email to hello@typst.app saying they approve this package (using their institutional email address, of course). If it is unofficial, I would like you to change the name for something more distinctive: if someone else makes an alternative to this package, they shouldn't be disadvantaged because you reserved the canonical name. Generally, people add an adjective to make their package name more unique, but you can get creative and make a pun or some indirect reference to the CTU if you want.